src/components/LibraryItem.tsx

```diff
@@ -28,11 +28,12 @@ export interface LibraryItemProps {
     showItemSummary: boolean,
     isLastItem?: boolean,
     onItemWillExpand?: Function
+    tooltipContent: any
 }
 
 export interface LibraryItemState {
     itemSummaryExpanded: boolean,
-    expanded: boolean
+    expanded: boolean,
```

Comment:
I dont think you need a ',' for the last property. Does this not fail?

Reply:
@reddyashish There are lots of typescript errors and it compiles anyway so I assume typescript is not set as strict. By the way according with typescript doc I think those commas should be semicolons.
Anyway I'm gonna fix this, thanks